### ISSUE TITLE
SAM PICKER VAC

### ISSUE THREAD

**Issue by turokman** [gibbed/SteamAchievementManager#315]
Is it possible to get a VAC ban for using the SAM Picker while playing csgo, but at the same time knocking out an achievement in another game that is not protected by the VAC server?

**Comment by gibbed** [comment]
I've never heard of it happening, but I can't make any guarantees, I'm not Valve.

The former FAQ suggested you shouldn't be running SAM while any game that uses VAC is running. I should probably port that FAQ over to GitHub.

**Comment by turokman** [comment]
Ye, it would be useful.

I saw that recommendation and ofc I knew it. I didn't it for purpose, just forgot for a moment. Now I am a little bit worried that I could get a VAC ban is CSGO, 'cause I've used SAM while I was playing it. But I hacked an achievement from the other game, not CSGO.

In that recommendation and a bunch of people in the Internet say that it might/can/may. And also a lot of people say that is safe 'cause SAM doesn't trigger VAC at all. IDK how this thing works, that's why I'm asking.

**Comment by gibbed** [comment]
As I said, I'm not Valve. I have no idea if they are or aren't doing anything so it's impossible for me to answer.

**Comment by turokman** [comment]
at least your program (you are a developer, I guess) does not affect the game files, especially csgo files, they did not give me any advantage. in addition, the achievement was opened in another game.

I read a lot of posts on reddit/steam today, some said they used sam while playing VAC-protected games and nothing happened.

I understand that you cannot give any guarantee, but I am encouraged by the fact that you say "should not" and not "prohibited".

Anyway, I hope everything is ok. getting a vac ban for hacking an achievement on a clean account would be incredibly stupid

Thank you!

**Comment by skurdab** [comment]
Well... I think you don't have to worry about this, I've tested unlock and lock again Counter-Strike Global Offensive's achievements to see if I can get VAC ban, its developed by Valve, it accesses VAC system of course. And I came come the conclusion that SAM won't trigger VAC ban. My testing account still doesn't get VAC ban or OW ban so far right now.

**Comment by turokman** [comment]
> Well... I think you don't have to worry about this, I've tested unlock and lock again Counter-Strike Global Offensive's achievements to see if I can get VAC ban, its developed by Valve, it accesses VAC system of course. And I came come the conclusion that SAM won't trigger VAC ban. My testing account still doesn't get VAC ban or OW ban so far right now.

Yes, mine too. I did it literally year agoband everything is fine.
